crates/re_viewer/src/ui/rerun_menu.rs

```diff
@@ -45,7 +45,7 @@ impl App {
                 // On the web the browser controls the zoom
                 let zoom_factor = self.app_options().zoom_factor;
                 ui.weak(format!("Zoom {:.0}%", zoom_factor * 100.0))
-                    .on_hover_text("The zoom factor applied on top of the OS scaling factor.");
+                    .on_hover_text("The zoom factor applied on top of the OS scaling factor");
```

Comment:
Suggestion: `The zoom level on top of the operating system's default value`

Reply:
I used `The UI zoom level....` as per above.